tpp-backend/desktop-image-experiment/desktop.md

```diff
@@ -0,0 +1,68 @@
+## Creating a Hyper-V image
```

Comment:
Is there an argument to renaming this file `README.md`? In either case, I think we need to link to this document prominently in the right place from the main repo README, and indicate where in the process this manual futzing must be done?

Reply:
I will delete this.

The information about building a Hyper-V image does not belong in the main project README imo.